### ISSUE TITLE
GitHub release for packaging?

### ISSUE THREAD

**Issue by palmskog** [jtassarotti/coq-proba#2]
Any chance of a GitHub release of this project? This would allow easy packaging in Coq's [OPAM repo](https://github.com/coq/opam-coq-archive) which leads to better exposure/visibility (and hopefully more external contributions).

**Comment by jtassarotti** [comment]
Yes, I would like to make one, after I finish and merge the `typeclasses` branch (hopefully soon)
